README.md

```diff
@@ -67,7 +68,9 @@ API 21+
 
 ### 1. Using Gradle
 
-- Our library is published through JCenter, so make sure the `jcenter()` repository has been added to your project level build.gradle file (which usually is automatic).
+- Our library is now published using MavenCentral, so make sure the `mavenCentral()` repository has been added to your project level build.gradle file if you want to consume versions `>= 0.0.17`.
+
+- If you are still using older versions you can consume `jcenter()` too. Please note it will only work for versions `<=0.0.16`
```

Comment:
Should it be <=0.0.10? because other versions from 10 to 16 we did not publish formally, right?

Reply:
Consumers can still access what is available publicly. So, just for consistencies have mentioned everything. But official releases can be found here https://github.com/microsoft/fluentui-android/releases